Code/Framework/AzToolsFramework/AzToolsFramework/ComponentMode/ComponentModeCollection.cpp

```diff
@@ -218,7 +218,7 @@ namespace AzToolsFramework
             // this call to activate the component mode editor state should eventually replace the bus call in
             // ComponentModeCollection::BeginComponentMode() to EditorComponentModeNotifications::EnteredComponentMode
             // such that all of the notifications for activating/deactivating the different editor modes are in a central location
-            m_viewportEditorModeTracker->ActivateMode({ /* DefaultViewportId */ }, ViewportEditorMode::Component);
+            m_viewportEditorModeTracker->ActivateMode({ GetEntityContextId() }, ViewportEditorMode::Component);
```

Comment:
comment: In future we'll ideally want to pass this through (ideally when `ComponentModeCollection` gets created we pass in the `EntityContextId` from the context that created it) but this is a bigger change and makes sense to happen as a separate PR, just wanted to mention it for future work 🙂

Reply:
:+1